### DISCUSSION TITLE
IOS widgets to display photos in Immich

### DISCUSSION THREAD

**Discussion by owpac** [immich-app/immich#11256] [General]
Hello

I would love to have widgets to use with Immich to be able to see my photos randomly.

Thanks for all the good work!

**Comment by radh21301** [comment]
Maybe similar to this?

https://github.com/immich-app/immich/discussions/6082

**Reply by owpac** [reply]
Oh yes, seems quite similar! Thanks, didn't find it before, I'm going to comment it.
